docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_cf.md

```diff
@@ -1,90 +1,90 @@
 
-Please see the following important points about this change:
+See the following points about this change:
 
--   Starting with CF196, versioning now implements a 3-digit integer that increments every release (CF196, CF197, CF198… and so on.)
--   Though the versioning is the same, the CFs are packaged separately and specific to each deployment. Please find the sample file naming for the CF package applicable to an on-premises deployment versus a container deployment.
-    -   On-premises: HCL-DX-CF196_Server_Update.zip, HCL-DX-CF196_Express_Update.zip, or HCL-DX-CF196_RemoteSearch_Update.zip.
-    -   Container: hcl-dx-kubernetes-v95-CF196.zip
--   CF196 and later releases are applicable to HCL DX 8.5, 9.0, and 9.5 on-premise deployments.
+-   Starting with CF196, versioning uses a 3-digit integer that increments every release (CF196, CF197, CF198, and so on).
+-   Although the versioning is the same, the CFs are packaged separately and specific to each deployment. You can find the sample file naming for the CF package that applies to an on-premises deployment versus a container deployment.
+    -   On-premises: `HCL-DX-CF196_Server_Update.zip`, `HCL-DX-CF196_Express_Update.zip`, or `HCL-DX-CF196_RemoteSearch_Update.zip`.
+    -   Container: `hcl-dx-kubernetes-v95-CF196.zip`
+-   CF196 and later releases apply to HCL DX 8.5, 9.0, and 9.5 on-premises deployments.
 -   Customers can update directly to the latest CF release without installing multiple individual CFs \(for example, 8.5 CF17 to 8.5 CF196).
 
-## Learning about this Combined Cumulative Fix
+## Learning about this CF
 
-Some combined cumulative fixes focus on fixing known issues that are identified by APARs. Other cumulative fixes are focused on new features.
+Some CFs focus on fixing known issues that are identified by APARs. Other CFs focus on new features.
 
-1.  Get an overview of the features and improvements delivered in the latest combined cumulative fix.
+<!-- The following list needs an introductory sentence. What's the nature of the items in this list? -->
 
-    -   Documentation resource: [Apply Combined Cumulative Fix](../../../../../deployment/install/traditional/cf_install/index.md)
-2.  View the fixes delivered for HCL Digital Experience 9.5. The documentation provides specific information on how to enable and use the new feature.
+1.  Get an overview of the features and improvements delivered in the latest combined cumulative fix: [Apply Combined Cumulative Fix](../../../../../deployment/install/traditional/cf_install/index.md) 
 
-    -   Documentation resource: [What's new with CF17](../../../../../whatsnew/cf17/new_cf17.md)
+    -   
+2.  View the fixes delivered for HCL Digital Experience 9.5. The documentation provides specific information about how to enable and use the new feature: [What's new with CF17](../../../../../whatsnew/cf17/new_cf17.md)
```

Comment:
View the fixes delivered for HCL DX 9.5. The documentation provides specific information about how to enable and use the new feature. For more information refer [What's new with CF17].

Reply:
For more information. refer to